### PR TITLE
Clean more Warnings, fix yuzu-emu/yuzu#252

### PR DESCRIPTION
**Pull request by N00byKing**

### CODE REVIEW

File: src/core/hle/service/filesystem/fsp_srv.cpp [hunk 1]
```diff
@@ -170,7 +170,7 @@ class IDirectory final : public ServiceFramework<IDirectory> {
         IPC::RequestParser rp{ctx};
         const u64 unk = rp.Pop<u64>();
 
-        LOG_DEBUG(Service_FS, "called, unk=0x%llx", unk);
+        LOG_DEBUG(Service_FS, "called, unk=0x%lx", unk);
```
Comment (MerryMage) [inline src/core/hle/service/filesystem/fsp_srv.cpp]: `LOG_DEBUG(Service_FS, "called, unk=0x%" PRIx64, unk);`

Comment (N00byKing) [reply src/core/hle/service/filesystem/fsp_srv.cpp]: I understand what you mean, but shouldnt it be 32?

Comment (MerryMage) [reply src/core/hle/service/filesystem/fsp_srv.cpp]: is `unk` not a `u64`?

**Comment by N00byKing** [comment]
its unsigned long not long long (thats where the warning came from)

File: src/core/hle/service/filesystem/fsp_srv.cpp (hunk 1 above)
Comment (N00byKing) [reply src/core/hle/service/filesystem/fsp_srv.cpp]: its unsigned long not long long (thats where the warning came from)

Comment (bunnei) [reply src/core/hle/service/filesystem/fsp_srv.cpp]: @MerryMage is right - %lx will only print a 32-bit value. `unk` is a 64-bit value. Applies here and elsewhere - @N00byKing can you please clean this up? I noticed in other places we are now logging 64-bit values as %lx, and therefore it's only printing the lower 32 bits.

**Comment by lioncash** [comment]
Can you please prefix your commits properly like how almost everyone else does it? i.e. prefixing the thing you're changing to the message.

e.g. If it's a general change to core, then `core: <thing done>` would be the message. If it's localized in some_file.cpp/.h, then it's `some_file: <thing done>`. It makes it easier to filter out (or find) commits in the git log (and provides context for changes without needing actually read the diff).

**Comment by N00byKing** [comment]
@lioncash will do

File: src/core/hle/service/filesystem/fsp_srv.cpp (hunk 1 above)
Comment (N00byKing) [reply src/core/hle/service/filesystem/fsp_srv.cpp]: Addressed

Comment (N00byKing) [reply src/core/hle/service/filesystem/fsp_srv.cpp]: The Logging Problems have been fixed, and I have changed the PR title to mention that

**Comment by N00byKing** [comment]
Fuck. I have no idea what just happened. Gimme a sec

**Comment by Hexagon12** [comment]
What's the status of this PR?

**Comment by N00byKing** [comment]
@Hexagon12 *should* be fine

**Comment by daniellimws** [comment]
Minor request here, could you update the logging to use the new macros? e.g. `NGLOG_INFO` instead of `LOG_INFO`.

**Comment by N00byKing** [comment]
Tbh, while I thought about it, I think it's out of scope for this PR. *After* it has been merged, maybe

**Comment by daniellimws** [comment]
Sure then

**Comment by N00byKing** [comment]
By now this PR is so outdated conflicts are pretty much everywhere. Closing

**Pull request closed without merging**